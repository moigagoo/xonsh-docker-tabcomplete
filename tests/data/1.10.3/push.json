{
        "arguments": [
                "--disable-content-trust=true",
                "--help"
        ],
        "commands": []
}
