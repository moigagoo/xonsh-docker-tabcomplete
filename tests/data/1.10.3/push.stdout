
Usage:	docker push [OPTIONS] NAME[:TAG]

Push an image or a repository to a registry

  --disable-content-trust=true    Skip image signing
  --help                          Print usage
